src/jobs/deploy.yml

```diff
@@ -38,5 +42,5 @@ steps:
       name: EB Deploy
       working_directory: <<parameters.app-dir>>
       command: |
-        eb init <<parameters.environment-name>> -r $AWS_DEFAULT_REGION -p <<parameters.platform-version>>
+        eb init <<parameters.application-name>> -r $AWS_DEFAULT_REGION -p <<parameters.platform-version>>
```

Comment:
For users that have modified the environment name, this will unfortunately be a breaking change.